spec/spec_helper.rb

```diff
@@ -0,0 +1 @@
+require "logstash/devutils/rspec/spec_helper"
```

Comment:
Good point.  We should put these lines in the example repos (as I followed that method).